### PR TITLE
How it works tbtc v2 updates

### PR DESCRIPTION
**Pull request by r-czajkowski**
Depends on: #329 

This PR updates the tBTC how it works page according to the Figma views.
<details>
  <summary>Screenshot</summary>
  
  obraz
  
</details>

### CODE REVIEW

File: src/pages/tBTC/HowItWorks/MintingTimelineCard.tsx [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+import { BsFillArrowRightCircleFill } from "react-icons/all"
+import {
+  MintingTimelineStep1,
+  MintingTimelineStep2,
+  MintingTimelineStep3,
+} from "../Bridge/MintingCard/MintingTimeline"
+import Link from "../../../components/Link"
+import { ExternalHref } from "../../../enums"
+
+export const MintingTimelineCard: FC<ComponentProps<typeof Card>> = ({
```
Comment (michalsmiarowski) [inline src/pages/tBTC/HowItWorks/MintingTimelineCard.tsx:20]: Minting timeline on How It Works page needs some polishing. This is how step 1 looks now:

image

Notice the `off-chain` label that is next to Step 1 (not above), image is centerd and also it looks like the description of the step is bigger on Figma. Similar differences are in other steps too

And this is how it should look based on Figma:

image

**Review by michalsmiarowski — CHANGES_REQUESTED** [review]
Overall looks good 👍 Left some comments to look at before the merge

File: src/pages/tBTC/HowItWorks/MintingTimelineCard.tsx (hunk 1 above)
Comment (r-czajkowski) [reply src/pages/tBTC/HowItWorks/MintingTimelineCard.tsx:20]: 8541d44 c831851

File: src/pages/tBTC/HowItWorks/AuditsCard.tsx [hunk 2]
```diff
@@ -2,12 +2,14 @@ import { FC, ComponentProps } from "react"
...
 
 export const AuditsCard: FC<ComponentProps<typeof Card>> = (props) => {
   return (
     <Card {...props}>
-      <LabelSm mb={5}>Audit Reports</LabelSm>
-      <List mt="5" spacing="6">
+      <LabelSm mb={8}>Audit Reports</LabelSm>
+      <List spacing="2">
         {contracts.map((contract) => (
           <DetailedLinkListItem {...contract} />
```
[the remark this answers is not in the document]
Comment (r-czajkowski) [reply src/pages/tBTC/HowItWorks/AuditsCard.tsx]: b918470db8d65bf9d5cae4b56b4fd7c27ec84c4a

File: src/enums/externalHref.ts [hunk 3]
```diff
@@ -22,4 +22,12 @@ export enum ExternalHref {
...
   randomBeaconNodeDocs = "https://docs.threshold.network/guides/threshold-applications/tbtc-v2-client-setup",
   analyticsReports = "SOME_URL",
+  // TODO: add correct URL address.
+  btcRecoveryAddress = "SOME_URL",
+  // TODO: add correct URL address.
+  tbtcBridgeGithub = "SOME_URL",
+  // TODO: add correct URL address.
+  tBTCBrdigeAudit = "SOME_URL",
+  // TODO: add correct URL address.
+  spearbitAudit = "SOME_URL",
```
[the remark this answers is not in the document]
Comment (r-czajkowski) [reply src/enums/externalHref.ts]: 85b6d23- updated `btcRecoveryAddress` and `tbtcBridgeGithub`. Waiting for other from designers.

**Review by michalsmiarowski — APPROVED** [review]
LGTM 🔥

File: src/enums/externalHref.ts (hunk 3 above)
[the remark this answers is not in the document]
Comment (r-czajkowski) [reply src/enums/externalHref.ts]: 91fb144

**Review by michalsmiarowski — APPROVED** [review]
LGTM v2 🔥

**Pull request merged**